test/benchgc/src/benchgc/pattern/mlp.py

```diff
@@ -220,13 +151,38 @@ def init_module(self, ctx: ir.Context) -> ir.Module:
                     func.ReturnOp([data])
         return module
 
-    def prepare_np_args(self, disable_results_to_params: False) -> List[np.ndarray]:
-        bench_func = get_kernel_func_from_module(self.ir_module, self.main_entry)
-        np_args = []
-        for arg in bench_func.arguments:
-            np_args.append(make_mlir_ndarray(arg.type))
-
-        if not disable_results_to_params:
-            for res in bench_func.type.results:
-                np_args.append(make_mlir_ndarray(res))
-        return np_args
+    def default_fill(
+        flags: argparse.Namespace,
+        arg: Arg,
+        arglist: List[Arg],
+    ):
+        layers = len(flags.hidden_size_list.strip().split("x"))
+        if arg.index == 0:
+            # src
+            arg.fill_type = "D"
+            arg.fill_param = [
+                "matmul",
+                "src",
+                arglist[0].dtype,
+                arglist[0].dtype,
+                arglist[0].dtype,
+                1,
+            ]
+        elif arg.index <= layers:
+            # wei
+            arg.fill_type = "D"
+            arg.fill_param = [
+                "matmul",
+                "wei",
+                arglist[0].dtype,
```

Comment:
shall we use different idx here for `wei`?

Reply:
Thank you for pointing this out. Previously, I tried to use a single matmul op's filling strategy in the MLP, but later I encountered some issues. Now, I have aligned the filling and compare strategies of the MLP with the MLP validation script in GC v1. In the future, a separate PR might be proposed to optimize the MLP's filling.